opus/application/static_media/css/opus.css

```diff
@@ -252,9 +252,10 @@ ul.searchMenu {
     border-width: 3px;
 }
 
-.modal {
+/* We have to comment this out to prevent user from clicking navbar link when modal is open */
```

Comment:
if we are commenting it out and no longer using it, why do we leave it here...?

Reply:
BTW, Dave and I were talking about this the other day. I think you added this in so the modal dialog would be slightly further down on the screen. But if it's there, then you can still click in the 4% region at the top, allowing you to do things like change tabs. If we want the modal to be further down, we need a different way to do it.